Orange/data/table.py

```diff
@@ -20,6 +20,9 @@
 
 from threading import Lock
 
+import tempfile
+import urllib
```

Comment:
Imports from the python standard library should be in the first group of imports (on top).